### PR TITLE
Fix refresh rate not changing bug

### LINKED ISSUE #182: Changing list refresh rate doesn't work
**Describe the bug**
Going in Settings - Anilist - Refresh rate I can change the list refresh rate, but whenever I go back to my list and again in settings, the settings will be reverted to the default value

**To Reproduce**
1. Edit refresh rate
2. Try manually refreshing
3. Go back to the home page
4. Go back to the refresh rate setting

**Expected behavior**
In step 4 the refresh rate should be the same one I set in step 1, but it's not. Instead, it is reverted to the default value (15).

**Desktop**
 - OS: Windows 10 Pro
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by NicoAiko**
Add limit on how low and high a refresh rate can be set
Update changelog
Update locales
Bump version to 0.7.8b

Fixes #182

### CODE REVIEW

File: src/assets/locales/es_MX.json [hunk 1]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "Iniciado sesión como {0}",
         "refreshRate": "Frecuencia de actualización",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (NicoAiko) [inline src/assets/locales/es_MX.json:176]: @DevonOrxi Could you provide a comment with the translation?
That'd be awesome!

File: src/assets/locales/id.json [hunk 2]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "Masuk sebagai {0}",
         "refreshRate": "Jarak penyegaran",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (nattadasu) [inline src/assets/locales/id.json:176]:
```suggestion
        "refreshRateRule": "Jarak penyegaran yang diperbolehkan berkisar antara 5 menit hingga 5 jam (5 - 300 menit)!",
```

File: src/assets/locales/my.json [hunk 3]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "Masuk sebagai {0}",
         "refreshRate": "Jarak penyegaran",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (nattadasu) [inline src/assets/locales/my.json:176]:
```suggestion
        "refreshRateRule": "Selang masa yang dibenarkan adalah dari 5 minit hingga 5 jam (5 - 300 minit)!",
```

File: src/assets/locales/ar.json [hunk 4]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "تسجيل الدخول بـ {0}",
         "refreshRate": "معدل التحديث",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (nattadasu) [inline src/assets/locales/ar.json:176]:
```suggestion
        "refreshRateRule": "الفترة معدل التحديث بها من 5 دقائق إلى 5 ساعات (5 - 300 دقيقة)!",
```

File: src/assets/locales/zh_CN.json [hunk 5]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "用户 {0} 已登录",
         "refreshRate": "刷新率",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json:176]:
```suggestion
        "refreshRateRule": "刷新时间间隔只可设置在 5 分钟到 5 小时（即 5 - 300 分钟）之间！",
```

File: src/assets/locales/zh_CN.json [hunk 6]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "用户 {0} 已登录",
         "refreshRate": "刷新率",
```
Comment (Mornwind) [inline src/assets/locales/zh_CN.json:175]:
```suggestion
        "refreshRate": "刷新频率",
```

File: src/assets/locales/fr.json [hunk 7]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "Connecté en tant que {0}",
         "refreshRate": "Taux de rafraîchissement",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (ExtazUxy) [inline src/assets/locales/fr.json:176]: L'intervalle autorisé est compris entre 5 minutes et 5 heures (5 - 300 minutes) !

File: src/assets/locales/es_MX.json (hunk 1 above)
Comment (DevonOrxi) [reply src/assets/locales/es_MX.json:176]: Sure thing!

`"refreshRateRule":  "El intervalo permitido va de 5 minutos a 5 horas (5 - 300 minutos)"`

File: src/assets/locales/it.json [hunk 8]
```diff
@@ -173,6 +173,7 @@
       "aniList": {
         "loggedInAs": "Entrato come {0}",
         "refreshRate": "Frequenza di ricaricamento",
+        "refreshRateRule": "The allowed interval reaches from 5 minutes to 5 hours (5 - 300 minutes)!",
```
Comment (Knocks83) [inline src/assets/locales/it.json:176]: "refreshRateRule": "L'intervallo consentito è tra 5 minuti e 5 ore (5 - 300 minuti)!",

Comment (NicoAiko) [reply src/assets/locales/it.json:176]:
```suggestion
        "refreshRateRule": "L'intervallo consentito è tra 5 minuti e 5 ore (5 - 300 minuti)!",
```

**Pull request merged**